src/operator/manager.go

```diff
@@ -77,11 +79,26 @@ func main() {
 	}
 	clientset := k8s.GetClientset(kubeConfig)
 
+	// Get K8s version.
+	k8sVersion := ""
+	discoveryClient, err := discovery.NewDiscoveryClientForConfig(kubeConfig)
+	if err != nil {
+		log.WithError(err).Error("Failed to get discovery client for config")
+	} else {
+		version, err := discoveryClient.ServerVersion()
+		if err != nil {
+			log.WithError(err).Error("Failed to get server version from discovery client")
+		} else {
+			k8sVersion = strings.TrimPrefix(version.GitVersion, "v")
```

Comment:
hmm, do we do this elsewhere too? I wonder if we should use [ParseTolerant](https://pkg.go.dev/github.com/blang/semver/v4#ParseTolerant) instead? seems a bit less brittle that way.

Reply:
We do trim the "v" in other places, but agree ParseTolerant is better. Going to change this PR to not modify the string, and use ParseTolerant in configmgr service.